src/emm/services/github_service.py

```diff
@@ -0,0 +1,61 @@
+"""Service for working with github CLI."""
+from pathlib import Path
+from typing import List, Optional
+
+import plumbum
+from plumbum import local
+
+
+class GithubService:
+    """A service for interacting with github CLI."""
+
+    def pull_request(self, extra_args: List[str], directory: Optional[Path] = None) -> str:
+        """
+        Create the pull request using Github CLI.
+
+        :param extra_args: Extra arguments to pass to the github CLI.
+        :param directory: Directory to execute command at.
+        :return: Url of the created pull request.
+        """
+        args = ["pr", "create"]
+        args.extend(extra_args)
+        with local.cwd(self._determine_directory(directory)):
+            return local.cmd.gh[args]().strip()
+
+    def pr_comment(self, pr_url: str, pr_links: str, directory: Optional[Path] = None) -> None:
```

Comment:
This looks like it is mixing concerns. The function at this level should have no idea what `pr_links` is. It should only care about making a comment on the PR. If in the future, we want to add a PR comment for another reason, we would ideally use this same function, which would make an argument called `pr_links` confusing.

Reply:
I renamed the **pr_links** to **comment**. The function itself actually only add a comment base on the given pr_url, maybe the **pr_links** make it confusing.